test/test_artifacts/v2/jupyter-idle-ext.test.Dockerfile

```diff
@@ -0,0 +1,6 @@
+ARG SAGEMAKER_DISTRIBUTION_IMAGE
+FROM $SAGEMAKER_DISTRIBUTION_IMAGE
+
+ARG MAMBA_DOCKERFILE_ACTIVATE=1
+
+CMD ["python", "-c", "import jupyter_idle"]
```

Comment:
Please add better tests than this. 
I'm assuming that your extension returns a json response. can we at least validate whether the response has the correct keys.

Reply:
Will add a functional test as a follow up later in the week.

Reply:
Added a new commit with unit test changes